frontend/app/routes/$lang+/_public+/apply+/$id+/adult/personal-information.tsx

```diff
@@ -379,65 +379,68 @@ export default function ApplyFlowPersonalInformation() {
         <Progress aria-labelledby="progress-label" value={60} size="lg" />
       </div>
       <div className="max-w-prose">
-        <p className="mb-6">{t('apply-adult:contact-information.form-instructions')}</p>
-        <p className="mb-6 italic">{t('apply:optional-label')}</p>
         {errorSummaryItems.length > 0 && <ErrorSummary id={errorSummaryId} errors={errorSummaryItems} />}
         <fetcher.Form method="post" noValidate>
           <input type="hidden" name="_csrf" value={csrfToken} />
-          <p id="adding-phone" className="mb-2">
-            {t('apply-adult:contact-information.add-phone')}
-          </p>
-          <div className="mb-6 grid items-end gap-6 md:grid-cols-2">
-            <InputField
-              id="phone-number"
-              name="phoneNumber"
-              className="w-full"
-              autoComplete="tel"
-              defaultValue={defaultState?.phoneNumber ?? ''}
-              errorMessage={errorMessages['phone-number']}
-              label={t('apply-adult:contact-information.phone-number')}
-              maxLength={100}
-              aria-describedby="adding-phone"
-            />
-            <InputField
-              id="phone-number-alt"
-              name="phoneNumberAlt"
-              className="w-full"
-              autoComplete="tel"
-              defaultValue={defaultState?.phoneNumberAlt ?? ''}
-              errorMessage={errorMessages['phone-number-alt']}
-              label={t('apply-adult:contact-information.phone-number-alt')}
-              maxLength={100}
-              aria-describedby="adding-phone"
-            />
-          </div>
-          <p id="adding-email" className="mb-2">
-            {t('apply-adult:contact-information.add-email')}
-          </p>
-          <div className="mb-6 grid gap-6 md:grid-cols-2">
-            <InputField
-              id="email"
-              name="email"
-              className="w-full"
-              autoComplete="email"
-              defaultValue={defaultState?.email ?? ''}
-              errorMessage={errorMessages['email']}
-              label={t('apply-adult:contact-information.email')}
-              maxLength={100}
-              aria-describedby="adding-email"
-            />
-            <InputField
-              id="confirm-email"
-              name="confirmEmail"
-              className="w-full"
-              autoComplete="email"
-              defaultValue={defaultState?.confirmEmail ?? ''}
-              errorMessage={errorMessages['confirm-email']}
-              label={t('apply-adult:contact-information.confirm-email')}
-              maxLength={100}
-              aria-describedby="adding-email"
-            />
-          </div>
+          <p className="mb-6 italic">{t('apply:optional-label')}</p>
```

Comment:
Instructions which apply to the entire form should be shown at the begging and placed prior to the form element. 

Move it before the ErrorSummary.